versioned_docs/version-latest/protocol/pulsar.mdx

```diff
@@ -41,12 +34,24 @@ Pulsar requires commodity hardware specs to operate. At a **minimum**, it is rec
 | SWAP     | 4GB              |
 | Storage  | 100GB SSD        |
 
-:::note SWAP Warning
-A swap is only necessary if you have a low amount of RAM. 
+:::caution
```

Comment:
Okay that's interesting. First of all, thanks for reviewing all of these changes! I realize now that there's a lot mushed into one pull request haha. 

About this change --  these warnings seemed totally irrelevant to me starting up on my consumer hardware, which is why I hid them. But I totally get wanting them displayed prominently so that the people that they are relevant to, have no chance of missing them. I will revert them back.

Reply:
You are spot on, thank you, Skyler!